lms/envs/common.py

```diff
@@ -2189,7 +2189,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'openedx.core.djangoapps.cookie_metadata.middleware.CookieNameChange',
 
     # Monitoring and logging for expected and ignored errors
```

Comment:
@Yagnesh1998: I know you said you found this, but I guess you haven't updated it yet.

Reply:
@robrap When I push code those changes come currently changes are in a local system. Thanks.